notifications/tests.py

```diff
@@ -56,3 +58,75 @@ def test_surge_translation(self):
                         self.aws_translator._fake_translation(original_value, lang, 'en') if lang != 'en' else original_value,
                     )
+
+class SurgeAlertFilteringTest(APITestCase):
+
+    def test_filtering(self):
+        region_1 = Region.objects.create(name=1)
+        region_2 = Region.objects.create(name=2)
+        country_1 = Country.objects.create(name='Atlantis', iso3='ATL', region=region_1)
+        country_2 = Country.objects.create(name='Neptunus', iso3='NPT', region=region_2)
+        molnix_tag_1 = MolnixTag.objects.create(
+            molnix_id=3,
+            name='OP-6700',
+            description='Middle East Crisis | MENA',
+            tag_type='regular',
+            tag_category='molnix_operation')
+        molnix_tag_2 = MolnixTag.objects.create(
+            molnix_id=5,
+            name='L-FRA',
+            description='French',
+            tag_type='language',
+            tag_category='molnix_language')
+        molnix_tag_3 = MolnixTag.objects.create(
+            molnix_id=17,
+            name='AMER',
+            description='Americas Region',
+            tag_type='regular',
+            tag_category='molnix_region')
+        surge_alert_1 = SurgeAlert.objects.create(
+            message='CEA Coordinator, Floods, Atlantis', atype=6, category=2, molnix_status='active', country=country_1)
+        surge_alert_1.molnix_tags.set([molnix_tag_1, molnix_tag_2])
+        surge_alert_2 = SurgeAlert.objects.create(
+            message='WASH Coordinator, Earthquake, Neptunus', atype=5, category=1, molnix_status='active', country=country_2)
+        surge_alert_1.molnix_tags.set([molnix_tag_1, molnix_tag_2])
+        surge_alert_2.molnix_tags.set([molnix_tag_1, molnix_tag_3])
+
+        # we have 2 SurgeAlerts with 2 different Countries: A, B
+        # Filtering for X, B gives 1; filtering for A, B gives both results.
+
+        response = self.client.get('/api/v2/surge_alert/?country__iso3__in=AAA,NPT').json()
+        self.assertEqual(response['count'], 1)
+        self.assertEqual(response['results'][0]['molnix_tags'][0]['name'], 'OP-6700')
+
+        response = self.client.get('/api/v2/surge_alert/?country__iso3__in=ATL,NPT').json()
+        self.assertEqual(response['count'], 2)
+        self.assertEqual(
+            {response['results'][0]['molnix_tags'][0]['name'],
+             response['results'][0]['molnix_tags'][1]['name'],
+             response['results'][1]['molnix_tags'][0]['name'],
+             response['results'][1]['molnix_tags'][1]['name']},
+            {'OP-6700', 'L-FRA', 'AMER'})
+
+        # we have 2 SurgeAlerts with MolnixTags A, B and A, C.
+        # Filtering for A, B, C gives both. Filtering for only B or C gives 1.
+
+        # filtering by molnix_tags
+        response = self.client.get('/api/v2/surge_alert/?country__iso3__in=ATL,NPT&molnix_tags=' + str(molnix_tag_2.id)
+            ).json()
```

Comment:
We can pass GET arguments as dict.
https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.Client.get

```suggestion
        def _to_csv(*items):
            return ','.join([str(i) for i in items])

        response = self.client.get(
            '/api/v2/surge_alert/'
            {
                'country__iso3__in': 'ATL,NPT',
                'molnix_tags': _to_csv(molnix_tag_2.id),
            },
        ).json()
```